template.html

```diff
@@ -1,117 +1,116 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <html lang="ja">
-<head>
-    <meta charset="UTF-8">
+  <head>
+    <meta charset="UTF-8" />
     <title>Docker ps</title>
     <style>
-        body {
-            margin: 0;
-        }
-        .header {
-            position: fixed;
-            top: 0;
-            left: 0;
+      body {
+          margin: 0;
+      }
+      .header {
+          position: fixed;
+          top: 0;
+          left: 0;
 
-            width: 100%;
+          width: 100%;
 
-            padding: 0.5em;
-            height: 3em;
+          padding: 0.5em;
+          height: 3em;
 
-            background-color: #4c526e;
-            color: #ddd;
-        }
-        .soft-title {
-            font-weight: bold;
-            font-size: 1.3em;
-        }
-        .container-count {
-            font-size: 0.8em;
-        }
-        .content-wrapper {
-            margin: 1em;
-            margin-top: 4.5em;
-        }
-        .info-wrapper {
-            display: block;
-        }
-        .info-frame {
-            display: inline-block;
+          background-color: #4c526e;
+          color: #ddd;
+      }
+      .soft-title {
+          font-weight: bold;
+          font-size: 1.3em;
+      }
+      .container-count {
+          font-size: 0.8em;
+      }
+      .content-wrapper {
+          margin: 1em;
+          margin-top: 4.5em;
+      }
+      .info-wrapper {
+          display: block;
+      }
+      .info-frame {
+          display: inline-block;
 
-            max-width: 30em;
-            margin: 0 auto;
-            padding: 1em 2em;
- 
-            font-family: monospace;
-            border: 2px solid #aaa;
+          max-width: 30em;
+          margin: 0 auto;
+          padding: 1em 2em;
 
-            background-color: #f0f0f0;
-         }
-        .container-id {
-            margin: 0.3em;
-            margin-top: 0em;
+          font-family: font-family: 'メイリオ', Meiryo,'Yu Gothic UI','ヒラギノ角ゴシック','Hiragino Sans','ヒラギノ明朝 ProN','Hiragino Mincho ProN',sans-serif;
```

Comment:
```suggestion
          font-family: 'メイリオ', Meiryo,'Yu Gothic UI','ヒラギノ角ゴシック','Hiragino Sans','ヒラギノ明朝 ProN','Hiragino Mincho ProN',sans-serif;
```
あと、Body全体にFont設定を適用しても良いのかもしれませんね。info-frameだけじゃなくて

Reply:
了解しました